README.md

```diff
@@ -40,6 +40,8 @@ After creating and broadcasting the transaction, save the `txid` to [out.txt](./
 
 ## Execution
 To test your solution locally:
+- Ensure that you have `npm` and `nvm` installed and your system `node -v` is at latest stable version.
```

Comment:
we should also mention that you need node v18 or greater

Reply:
Done..